utils/validate.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python2.7
```

Comment:
Would be great to have Python 3 support for new scripts.

Reply:
it seems to me that https://github.com/EFForg/https-everywhere-docker-base does not support Python 3, @Hainish can you confirm this?